pycbc/events/stat.py

```diff
@@ -70,12 +73,15 @@ def __init__(self, files=None, ifos=None, **kwargs):
 
         self.ifos = ifos or []
 
+        self.sngl_ranking = sngl_ranking
+        self.sngl_ranking_kwargs = {}
+        for key, value in kwargs.items():
+            if key.startswith('sngl_ranking_'):
+                self.sngl_ranking_kwargs[key[13:]] = value
 
-class NewSNRStatistic(Stat):
-    """Calculate the NewSNR coincident detection statistic"""
-
-    def single(self, trigs):
-        """Calculate the single detector statistic, here equal to newsnr
+    def get_sngl_ranking(self, trigs):
```

Comment:
to be consistent with other suggestions, this stuff would rather be get_sngl_stat.